### PR TITLE
updates kubeless-ui image to fix subpath route

### PR DESCRIPTION
**Pull request by prydonius**

### CODE REVIEW

File: kubeless-ui.jsonnet [hunk 1]
```diff
@@ -33,7 +33,9 @@ local host = "kubeless-ui";
           serviceAccountName: $.serviceAccount.metadata.name,
           containers_+: {
             default: kube.Container("ui") {
-              image: "bitnami/kubeless-ui:v0.2.3",
+              // FIXME: point to real tag
+              image: "bitnami/kubeless-ui:latest",
+              imagePullPolicy: "Always",
```
Comment (ngtuna) [inline kubeless-ui.jsonnet]: should we wait for the new release `v0.2.4` ?

Comment (prydonius) [reply kubeless-ui.jsonnet]: I want to get this in front of people so they can test everything works, I created #15 to revisit this as soon as we have some validation.

Comment (anguslees) [inline kubeless-ui.jsonnet]: Remove `imagePullPolicy` - the k8s default is already exactly what we want: `Always` if `:latest`, else `IfNotPresent`

**Review by anguslees — CHANGES_REQUESTED** [review]

File: kubeless-ui.jsonnet (hunk 1 above)
Comment (prydonius) [reply kubeless-ui.jsonnet]: good point, done

**Review by anguslees — APPROVED** [review]

**Pull request merged**